docs/openapi/schemas.yaml

```diff
@@ -1517,3 +1520,91 @@ ImportAssistantResponse:
     choices:
       - message:
           content: '.breadcrumbs'
+ApiKeyRequest:
+  type: object
+  required:
+    - name
+    - feature
+    - domains
+  properties:
+    name:
+      description: The name of the API key
+      type: string
+    feature:
+      description: The feature the API key is for
+      type: string
+    domains:
+      description: The domains the API key is allowed to access
+      type: array
+      items:
+        type: string
+  example:
+    name: 'Example API Key'
+    feature: 'imports'
+    domains:
+      - 'example.com'
+ApiKeyResponse:
+  type: object
+  properties:
+    id:
+      description: The ID of the API key
+      $ref: '#/Id'
+    name:
+      description: The name of the API key
+      type: string
+    apiKey:
+      description: The API key
+      type: string
+    scopes:
+      description: The scopes of the API key
+      type: array
+      items:
+        type: string
+    domains:
+      description: The domains the API key is allowed to access
```

Comment:
If the above `scopes` is an array of objects, then `imports.write` (or any other scope which needs it) will contain the domains information, so it wouldn't need to be duplicated here.

Reply:
I was thinking about it - should we keep it consistent everywhere? Move `domains` to `scopes` everywhere. Or move `domains` to a higher level everywhere (even in the database).